src/main/java/service/FileService.java

```diff
@@ -0,0 +1,9 @@
+package service;
+
+import java.util.List;
+
+public interface FileService {
+    List<String> readFromFile(String filename);
+
+    void writeToFile(List<String> report, String toFileName);
```

Comment:
```suggestion
    void writeToFile(List<String> lines, String toFileName);
```
Remember about abstraction. This interface can be used for many things, not for reports only.